terraform/Using_Terraform.md

```diff
@@ -13,20 +13,20 @@ The Makefile contain all the commands to use the Terraform, however if you'd lik
 3. Create the file `my_variables.auto.tfvars` with the following Terraform input variables using your own specific values:
 
    ```hcl
-   owner                        = "johandry"
+   owner                        = "bob"
    project_name                 = "cloud-pak-app"
-   entitled_registry_user_email = "Johandry.Amador@ibm.com"
+   entitled_registry_user_email = "bob@email.com"
    ```
 
-   Append the input variable `cluster_id` if you have an existing Openshift cluster to install the Cloud Pak, like so:
+   Append the input variable `cluster_id` if you have an existing Openshift cluster to install the Cloud Pak on, like so:
 
    ```hcl
-   cluster_id                   = "bupkhgbd00id2uu15cjg"
+   cluster_id                   = "xxxxxxxxxxxxxxxxxxxxx"
    ```
 
-   Open the file `terraform.tfvars` or `variables.tf` and feel free to overwrite the input default values for any of the existing variables.
+   Open the file `cp4xxx/terraform.tfvars` or `variables.tf` to verify or overwrite the input default values for any of the existing variables. For more information on each Cloud Pak's specific inputs go to [cp4mcm](cp4mcm/README.md), [cp4app](cp4app/README.md) or [cp4data](cp4data/README.md).
 
-4. Until the permissions issue is not solved you need to provide the VLANs. Execute the command `ibmcloud ks vlan ls --zone {datacenter}`, get a private and public VLAN, and write them down in the `terraform.tfvars` file. Example:
+   NOTE: Until the permissions issue is solved you need to provide the VLANs. Execute the command `ibmcloud ks vlan ls --zone {datacenter}`, get a private and public VLAN, and store them in the `cp4xxx/terraform.tfvars` file. Example:
```

Comment:
Similar comment like in the other document, this is a paragraph here (which is fine). In the other document it is a list item. 
I'd change them both to the same format: paragraph or list item.

Reply:
Also, remove `cp4xxx`, the user is already in that directory.